frontend/src/scenes/pipeline/types.ts

```diff
@@ -1,6 +1,6 @@
 import {
     BatchExportConfiguration,
-    BatchExportDestination as BatchExportService,
```

Comment:
As evidenced here, `BatchExportDestination` being an overloaded term was already an issue before, so we called this "Service" here for disambiguation. Based on your comment here, Tiina, I made this disambiguation global.